webpack.config.js

```diff
@@ -1,10 +1,9 @@
 /* eslint-env node */
 
 /* TODO:
-- watch
 - hot module reloading
 - don't export globals (bundle to read data from template) https://github.com/webpack/webpack/issues/2683#issuecomment-228181205
-- once rollup is gone update to babel-loader@8 and @babel/core, etc
```

Comment:
Tried updating to `babel-loader@8` and `@babel/code@7`, etc, but found it more problematic then helpful.

First issue: swiper `.esm` modules didn't transpile correctly for some reason. I managed to find a workaround solution by copying babelrc config into webpack config.
But it seems that the bundles built with `babel@7` are slightly larger then those we currently have with `babel@6`, so I guess it's ok to stay on a bit older version.